base/util.go

```diff
@@ -139,6 +139,10 @@ func ToInt64(value interface{}) (int64, bool) {
 	return 0, false
 }
 
+func IsPseudoDoc(ID string) bool {
+	return strings.HasPrefix(ID, "_user/") || strings.HasPrefix(ID, "_role/")
+}
+
```

Comment:
Since this is changes-specific functionality - I think it makes more sense for this method to live in changes.go.